js/server/upgrade-database.js

```diff
@@ -754,13 +750,43 @@
         });
       }
     });
+    addTask({
+      name: 'createJobsIndex',
+      description: 'create index on attributes in _jobs collection',
+
+      system: DATABASE_SYSTEM,
+      cluster: [CLUSTER_NONE, CLUSTER_COORDINATOR_GLOBAL],
+      database: [DATABASE_INIT, DATABASE_UPGRADE],
+
+      task: function () {
+        const tasks = getCollection('_jobs');
+
+        if (!tasks) {
+          return false;
+        }
+
+        tasks.ensureIndex({
+          type: 'skiplist',
+          fields: ['queue', 'status', 'delayUntil'],
+          unique: true,
+          sparse: false
+        });
+        tasks.ensureIndex({
```

Comment:
Why two indexes which are almost identical? Are they both needed because queries we run on the collection?